doc/rbd/rbd-snapshot.rst

```diff
@@ -42,18 +45,22 @@ Cephx Notes
 When `cephx`_ authentication is enabled (it is by default), you must specify a
 user name or ID and a path to the keyring containing the corresponding key. See
 :ref:`User Management <user-management>` for details. You may also set the
-``CEPH_ARGS`` environment variable to avoid re-entry of these parameters. ::
+``CEPH_ARGS`` environment variable to avoid re-entry of these parameters:
+
+.. prompt:: bash $
 
 	rbd --id {user-ID} --keyring=/path/to/secret [commands]
 	rbd --name {username} --keyring=/path/to/secret [commands]
```

Comment:
Let's make this `--keyring /path/to/secret` (without `=`), so that it looks like all other options.